core/src/trezor/ui/layouts/delizia/__init__.py

```diff
@@ -336,14 +342,14 @@ def show_danger(
     )
 
 
-def show_success(
+async def show_success(
     br_name: str | None,
     content: str,
     subheader: str | None = None,
     button: str | None = None,
     time_ms: int = 0,
-) -> Awaitable[None]:
-    return raise_if_not_confirmed(
+) -> None:
+    return await raise_if_not_confirmed(
```

Comment:
please revert this chunk, it (a) does not have any observable effect and (b) instead of `async -> return await` we prefer `def -> Awaitable`
(because calling an async function allocates an item, but passing an awaitable via a normal function doesn't)

Reply:
TIL - thanks!

Reply:
https://github.com/trezor/trezor-firmware/pull/5115/commits/176f0a69a888582937228f4a6dabf0b907eecc5d.